inferno-framework/us-core-test-kit
# Reference validation with version

# Summary
This PR fixes two issues
* attach version to target profile while doing reference validation 
* temporarily remove validation error for Patient's genderIdentity extension due to backward compatibility issue in US Core 6 ballot version. This filter will be removed after US Core 6 final release.

# Testing Guidance
* Verify that Patient test in US Core 5 and 6 does not return validation errors related to genderIdentify extension
* Verify that when using production reference server, Observation SDOH test in US Core 5 test kit passes
